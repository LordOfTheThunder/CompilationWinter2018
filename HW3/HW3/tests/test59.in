int TestRelop() {
	if (33 < 33b) {
		int k;
	}
	if (33b == 33) {
		int k;
	}
	if (33 < "potato") {
		int k;
	}
}



void main() { print("I hate this course"); }



