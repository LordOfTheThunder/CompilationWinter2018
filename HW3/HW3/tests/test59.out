---end scope---
k INT 0
---end scope---
---end scope---
k INT 0
---end scope---
line 8: type mismatch
